content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -30,7 +30,7 @@ This guide outlines the requirements of each policy.
 typically aimed at system- and infrastructure-level workloads managed by privileged, trusted users.
 
 The Privileged policy is defined by an absence of restrictions. Allow-by-default
-mechanisms (such as gatekeeper) may be Privileged by default. In contrast, for a deny-by-default mechanism (such as Pod
+mechanisms (such as Gatekeeper) may be Privileged by default. In contrast, for a deny-by-default mechanism (such as Pod
 Security Policy) the Privileged policy should disable all restrictions.
```

Comment:
It's sort of weird to reference gatekeeper and the now defunct PSP here. Maybe just remove those 2 sentences instead?

Reply:
I agree; remove them. Maybe:
```markdown
The Privileged policy is defined by an absence of restrictions. If you define a Pod where the Privileged
security policy applies, the Pod you define is able to bypass typical container isolation mechanisms.
For example, you can define a Pod that has access to the node's host network.
```

Reply:
Thanks - Applied